packages/CatalogService/__tests__/CatalogService.test.ts

```diff
@@ -0,0 +1,107 @@
+import {  CatalogService, DCATCatalog, DCATDataset, DCATResource, DCATDataService } from "../index";
+const cs = new CatalogService("http://localhost:3030/", "catalog_test", undefined, undefined,true);
+const rdfsResource = "http://www.w3.org/2000/01/rdf-schema#Resource";
+const testDefaultNamespace = "http://telicent.io/data/";
+const id1 = `${testDefaultNamespace}cat1`;
+const id2 = `${testDefaultNamespace}dataset1`;
+const id3 = `${testDefaultNamespace}dataservice1`;
+
+const initialTripleCount = 11
+const initialNodeCount = 3
+
+function delays(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+
+describe("CatalogService", () => {
+
+  beforeAll(async () => {
+    cs.runUpdate(["DELETE WHERE {?s ?p ?o }"]); //clear the dataset
```

Comment:
Might be worth trying this
```diff
-    cs.runUpdate(["DELETE WHERE {?s ?p ?o }"]); //clear the dataset
+    await cs.runUpdate(["DELETE WHERE {?s ?p ?o }"]); //clear the dataset
-    await delays(1000)
```

Reply:
That will work, that is what I did in the tests for the other services